scripts/github/TESTING_GITHUB_ACTIONS.md

```diff
@@ -0,0 +1,36 @@
+[//]: # (SPDX-FileCopyrightText: 2023 Lifely
+[//]: # (SPDX-License-Identifier: EUPL-1.2+)
+
+# Testing GitHub Actions using ACT
```

Comment:
Nice readme. :-) I do wonder if we will ever use this. Using ACT is quite a bit of work and in my experience from past projects we will probably never (again) use this nor read this readme. But let's see how it goes. :-)

Reply:
Yes, ACT is a bit of work, and it is not as easy as one would hope. I was looking for a way to test my changes to the scripts without having to merge it to `main`. ACT is one tool that can do this, and it seems to work OK.